src/sentry/incidents/logic.py

```diff
@@ -536,6 +536,8 @@ def create_alert_rule(
 
     :return: The created `AlertRule`
     """
+    from sentry.workflow_engine.migration_helpers.alert_rule import migrate_alert_rule
```

Comment:
circular import :/

Reply:
🤔 i think that the root of the issue is how much stuff is defined in this file cause of that (and it'd probably really difficult to untangle) - we could make a new method outside of this file that imports both `migrate_alert_rule` and `create_alert_rule` - but that'd mean we'd have to update all imports for `create_alert_rule`.

if `create_alert_rule` is really just a used in a couple APIs / tests, then we could just import `migrate_alert_rule` in the APIs, then we can delete all the create_alert_rule code when we are cleaning up ACI.

Reply:
👍 I moved it to the serializer where we call `create_alert_rule`. The other main place is in the test fixture but I'm not sure we need to have it there.